### ISSUE TITLE
[Question] Can I link an Elixir community client?

### ISSUE THREAD

**Issue by viniarck** [infinyon/fluvio-website#429]
Hi, All. 

I've built [fluvio-ex](https://github.com/viniarck/fluvio-ex), an Elixir client, I plan to maintain it for the foreseeable future (and it's released under Apache 2 license). 

Do you think I could link this project as a community client? Let me know.

I've also stress tested this client with 1M+ records, the experience using Fluvio Cloud was very smooth and convenient.

**Comment by ajhunyady** [comment]
@viniarck this is cool! Thank you for your contribution. @tjtelan is currently restructuring the docs, and the best person to coordinate with.

**Comment by viniarck** [comment]
> @viniarck this is cool! Thank you for your contribution. @tjtelan is currently restructuring the docs, and the best person to coordinate with.

Thanks, @ajhunyady. Got it. @tjtelan, much appreciated if you could ping me in the future when this part is completed.

**Comment by tjtelan** [comment]
@viniarck I have created a quick example page for your Elixir client with links [in this PR](https://github.com/infinyon/fluvio-website/pull/444).

I'm not aware if you had any of your own ideas for this page, but the current restructuring the API docs is lower priority to other areas. I wanted to give you an opportunity to see it first and provide feedback before anything gets committed to the public site. Most importantly, I wanted to make sure attribution for your work meets your expectations.

I only copied some of your examples and provided my interpretation of what I believe the code is doing. I'm not at all familiar with Elixir, so feel free to let me know if I've got it wrong or if you have any of your own suggestions.
